File: node/secretstore/decrypt.js
const SecretStore = require('@oceanprotocol/secret-store-client').default
const { Logger } = require('@oceanprotocol/squid')
const config = require('../config')
const input = require('../input');

(async () => {
    const secretStore = new SecretStore(config)

    try {
        const decryptedDocument = await secretStore.decryptDocument(input.docId, input.encryptedDocument)

        Logger.log('__result__', decryptedDocument)
    } catch (err) {
        Logger.error(err)
    }
})()

File: python/secretstore/decrypt.py
import configparser
from secret_store_client.client import Client
from ..input import INPUT
from ..config import CONFIG_FILE

config = configparser.ConfigParser()
config.read(CONFIG_FILE)
secret_store = Client(secret_store_url=config['keeper-contracts']['secret_store.url'],
                      parity_client_url=config['keeper-contracts']['parity.url'],
                      address=config['keeper-contracts']['parity.address'],
                      password=config['keeper-contracts']['parity.password'])
decrypted_document = secret_store.decrypt_document(INPUT['docId'], INPUT['encryptedDocument'])

print("__result__{}".format(decrypted_document))
